package.json

```diff
@@ -7,5 +7,25 @@
   },
   "devDependencies": {
     "standard": "^13.0.2"
+  },
+  "standard": {
+    "globals": [
```

Comment:
Is it worth saying where this list came from (in the commit message)?

Reply:
🤔 I'm not sure if it's worth it.

Most of these are to ignore Jasmine functions (i.e describe, it, assert etc) the others were really just globals refer to in various Javascript files that failed linting because they were not declared in the scripts themselves.